container-scan/action.yaml

```diff
@@ -1,64 +1,66 @@
 name: "Security Scan Composite Action"
-description: "GitHub Action for scanning container image for vulnerabilities using Lacework"
+description: "GitHub Action for scanning container images for vulnerabilities using Lacework."
 
 inputs:
   dockerhub-user:
     required: false
     default: ""
-    description: username for dockerhub
+    description: DockerHub username for authentication.
   dockerhub-password:
     required: false
     default: ""
-    description: password for dockerhub
+    description: DockerHub password for authentication.
   docker-config-file:
     required: false
-    description: Path to the docker config file (defaults to .docker-config.json) Must contain imageName, may contain dockerfile
+    description: Path to the Docker config file (default is .docker-config.json) Must contain imageName and optionally the Dockerfile path.
     default: .docker-config.json
   github-token:
     required: true
-    description: GitHub token
+    description: GitHub token for accessing repositories.
   lw-account-name:
     required: true
-    description: Lacework account name
+    description: Lacework account name for scanning.
   lw-access-token:
     required: true
-    description: Lacework access token
+    description: Lacework access token for scanning.
   image-name:
     required: false
-    description: Docker image name
+    description: Name of the Docker image to scan.
   image-tag:
     required: true
-    description: Docker image tag
+    description: Tag of the Docker image to scan.
   image-platform:
-    description: Target platform to build image for (eg. linux/amd64 (default), linux/arm64, etc)
     required: false
     default: linux/amd64
-
+    description: Target platform for Docker image (e.g., linux/amd64, linux/arm64).
   build-args:
     required: false
-    description: List of build arguments for docker build as key-value pairs (e.g., KEY=VALUE)
+    description: Build arguments for the Docker build as key-value pairs.
     default: ""
   secrets:
     required: false
-    description: List of secrets for docker build as key-value pairs (e.g., SECRET_KEY=VALUE)
+    description: Secrets for the Docker build as key-value pairs.
     default: ""
   enable-docker-build:
     required: false
     default: true
-    description: Docker image tag
+    description: Whether to enable Docker image building (default is true).
 
 outputs:
-  comment-id:
-    description: Comment ID of the test report
-    value: ${{ steps.comment-pr.outputs.comment-id }}
+  image-name:
+    description: Name of the scanned Docker image.
+  image-tag:
```

Comment:
Image tag is already a required input to the action so this one is technically not needed